README.md

```diff
@@ -11,6 +11,9 @@ For now, it supports only older models that use the HTTP protocol for communicat
 
 ## Installation
 
+!!!! BREAKING CHANGE !!!!
+Addition _of _http to platform name to allow parallel installation next to COAP component
+
```

Comment:
I don't think this is necessary. We'll announce the breaking change in the Release notes / Change log. I don't think that many people have installed this yet.